examples/imagenet/train_imagenet.py

```diff
@@ -271,6 +272,12 @@ def train_loop():
         if train:
             optimizer.zero_grads()
             loss, accuracy = model.forward(x, y)
+            if not graph_generated:
```

Comment:
It was debugging at first. But after offline discussion I come to think it is OK to generate graph in imagenet example.